protocol/linkpreview_unfurler_opengraph.go

```diff
@@ -58,6 +89,13 @@ func (u *OpenGraphUnfurler) Unfurl() (*common.LinkPreview, error) {
 		return preview, fmt.Errorf("failed to parse OpenGraph data")
 	}
 
+	faviconPath := GetFavicon(bodyBytes)
+	t, err := fetchThumbnail(u.logger, u.httpClient, faviconPath)
```

Comment:
Nitpick: the function name `fetchThumbnail` could be renamed. Something like `fetchImage` would be more appropriate because `thumbnail` isn't a word used as a synonym for `favicon`.